### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by amyspark-ng** [sam-astro/Z-Sharp#35] [Q&A] [answered]
- Does it support arrays or any type of list? If so how does it work
- Is there any discord server or some type of thing where i can ask for help or talk about things regarding the language?

**Comment by sam-astro** [comment, marked as answer]
There are no arrays/lists, this was going to be a feature but since I stopped working on this project, that will never be implemented. I have a discord server (https://discord.gg/9p82dTEdkN) for my entire channel, but please don't ask too many questions regarding Z# since I've long been done with the project and am also pretty rusty haha

**Reply by amyspark-ng** [reply]
Oh that's okay! don't worry, still thanks for the answer tho :)

**Reply by pmo6** [reply]
Is there at least a way to use pointer arithmetic or use vectors? Though I do understand that one could easily write a linked list class.
